### PR TITLE
fix(tests): export dataset tests fails with presto

### PR DESCRIPTION
**Pull request by dpgaspar**
### SUMMARY
Fix/Skip flaky test on presto for datasets export

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: tests/datasets/api_tests.py [hunk 1]
```diff
@@ -824,6 +824,8 @@ def test_export_dataset(self):
         Dataset API: Test export dataset
         """
         birth_names_dataset = self.get_birth_names_dataset()
+        if birth_names_dataset.database.backend == "presto":
```
Comment (bkyryliuk) [inline tests/datasets/api_tests.py:827]: maybe add a todo / git issue to fix it ?

Comment (dpgaspar) [reply tests/datasets/api_tests.py:827]: that's fast @bkyryliuk :)

Still want to see it fail, so doing the `pass`

Comment (dpgaspar) [reply tests/datasets/api_tests.py:827]: Did some debuging, take a look at `36f0907` commit, where I raise to see the dump from presto metadata. It's really strange it should work.

Comment (bkyryliuk) [reply tests/datasets/api_tests.py:827]: :) try https://github.com/apache/incubator-superset/blob/master/tox.ini#L37 for the local development ?

**Review by mistercrunch — APPROVED** [review]

**Pull request merged**